community/pike/APKBUILD

```diff
@@ -0,0 +1,200 @@
+		--without-GTK1 \
+		--with-GTK2 \
+		--with-jpeglib \
+		--without-Kerberos \
+		--without-msql \
+		--without-mysql \
+		--without-Odbc \
+		--without-oracle \
+		--with-GL \
+		--with-GLUT \
+		--with-_Regexp_PCRE \
+		--without-libpdf \
+		--without-sass \
+		--with-sane \
+		--with-SDL \
+		--with-SDL_Mixer \
+		--with-svg \
+		--with-tifflib \
+		--without-freetype \
+		--without-vcdiff \
+		--with-_Image_WebP \
+		--with-zlib"
+	PATH="${builddir}/bin:${PATH}" make doc || die "doc failed"
+}
+
+package() {
+	cd "$builddir"
+	for pkg in $(printf "%s" "$subpackages" | tr ' ' '\n'); do
+		depends="$depends $pkg=$pkgver-r$pkgrel"
+	done
+	make INSTALLARGS="--traditional" buildroot="$pkgdir" install
+}
+
+_move_item() {
+	local bn=$(basename "$1")
+	local dn=$(dirname "$1")
+	local dst="$2/$dn"
+	mkdir -p "$dst"
+	mv "$1" "$dst/$bn"
+}
+
+_install() {
+	local bn=$(basename "$1")
+	local dn=$(dirname "$1")
+	find "$dn" -maxdepth 1 -name "$bn" | \
+		while IFS= read item; do _move_item "$item" "$2"; done
+}
+
+bzip2() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname bzip2 support"
+	depends="pike-core=$pkgver-r$pkgrel bzip2"
+	_install "usr/lib/pike/modules/*Bz2.*" "$subpkgdir"
+}
+
+gdbm() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname GDBM support"
+	depends="pike-core=$pkgver-r$pkgrel gdbm"
+	_install "usr/lib/pike/modules/*Gdbm.*" "$subpkgdir"
+}
+
+gl() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname OpenGL/GLUT support"
+	depends="pike-core=$pkgver-r$pkgrel freeglut mesa-gl libsm"
+	_install "usr/lib/pike/modules/*GL*" "$subpkgdir"
+}
+
+gtk() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname GTK2 support"
+	depends="pike-image=$pkgver-r$pkgrel libglade gtk+2.0"
+	_install "usr/lib/pike/modules/*GTK2*" "$subpkgdir"
+	_install "usr/lib/pike/modules/Tools.pmod/PV.pike" "$subpkgdir"
+}
+
+image() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname image handling modules"
+	depends="pike-core=$pkgver-r$pkgrel giflib-utils libjpeg tiff libwebp"
+	_install "usr/lib/pike/?.?/modules/Image.pmod" "$subpkgdir"
+	_install "usr/lib/pike/modules/Image.so" "$subpkgdir"
+	_install "usr/lib/pike/modules/*Image_[!S]*.so" "$subpkgdir"
+	_install "usr/lib/pike/modules/*Image*.pmod" "$subpkgdir"
+	_install "usr/lib/pike/modules/Graphics.pmod" "$subpkgdir"
+	_install "usr/lib/pike/modules/Colors.pmod" "$subpkgdir"
+	_install "usr/lib/pike/modules/Protocols.pmod/X.pmod/XImage.pmod" "$subpkgdir"
+}
+
+pcre() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname PCRE support"
+	depends="pike-core=$pkgver-r$pkgrel pcre"
+	_install "usr/lib/pike/modules/*Regexp_PCRE.*" "$subpkgdir"
+}
+
+sdl() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname SDL support"
+	depends="pike-image=$pkgver-r$pkgrel sdl sdl_mixer"
+	_install "usr/lib/pike/modules/*SDL.*" "$subpkgdir"
+}
+
+svg() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname SVG support"
+	depends="pike-core=$pkgver-r$pkgrel librsvg"
+	_install "usr/lib/pike/modules/_Image_SVG.so" "$subpkgdir"
+}
+
+sqlite() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname SQLite support"
+	depends="pike-core=$pkgver-r$pkgrel sqlite-libs"
```

Comment:
`sqlite-libs` is pulled in automatically. There is no need to list it in depends.